### PR TITLE
fix(list-view): fix list entry css and tree view css

### PR DESCRIPTION
**Pull request by SMahil**
list entry and tree view css fixes

### CODE REVIEW

File: src/app/components/iterations-panel/iterations-panel.component.less [hunk 1]
```diff
@@ -224,7 +224,7 @@
     padding-left: 0;
   }
   .list-pf-chevron {
-    margin-right: 5px;
+    margin-right: 0px;
```
Comment (mindreeper2420) [inline src/app/components/iterations-panel/iterations-panel.component.less]: remove the `px`

File: src/app/components/work-item-list-entry/work-item-list-entry.component.html [hunk 2]
```diff
@@ -50,8 +53,8 @@
                 [allowDelete]="false"
                 (onLabelClick)="labelClick($event)"></f8-label>
             </div>
-            <div class="f8-wi__list-description">
-              <span class="color-grey pull-left"
+            <div class="f8-wi__list-description width-100px">
```
Comment (mindreeper2420) [inline src/app/components/work-item-list-entry/work-item-list-entry.component.html]: Why not just add the `width: 100px;` to the `.f8-wi__list-description` class? This is an odd additional class to have here and will require HTML to be updated for what should just be a CSS update.

File: src/app/components/work-item-list-entry/work-item-list-entry.component.less [hunk 3]
```diff
@@ -89,20 +89,21 @@
...
-    padding: 0 20px;
+    padding: 0 15px;
     vertical-align: middle;
     text-align: left;
   }
 }
 
-.width-10 {
-  width: 10%;
+.width-100px {
```
Comment (mindreeper2420) [inline src/app/components/work-item-list-entry/work-item-list-entry.component.less]: See comment above about adding the `width: 100px;` to the existing class on the element.

File: src/app/components/work-item-list-entry/work-item-list-entry.component.html [hunk 4]
```diff
@@ -7,19 +7,21 @@
...
 	<!-- info area -->
   <div class="list-view-pf-main-info">
-    <div class="list-view-pf-left type f8-wi__list-witype">
-      <span class="pull-left">
+    <div class="list-view-pf-left type pull-left f8-wi__list-witype">
+      <span class="pull-left margin-0">
         {{workItem.attributes['system.number']}}
       </span>
-      <span class="color-grey pull-left fa
+      <span class="color-grey pull-left margin-h-10
```
Comment (mindreeper2420) [inline src/app/components/work-item-list-entry/work-item-list-entry.component.html]: What does `margin-h-10` refer too? Is this controlled in some variable file, or is it a custom class?

File: src/app/components/work-item-list-entry/work-item-list-entry.component.html [hunk 5]
```diff
@@ -7,19 +7,21 @@
 <div class="f8-wi__list-entry">
 	<!-- info area -->
   <div class="list-view-pf-main-info">
-    <div class="list-view-pf-left type f8-wi__list-witype">
-      <span class="pull-left">
+    <div class="list-view-pf-left type pull-left f8-wi__list-witype">
+      <span class="pull-left margin-0">
```
Comment (mindreeper2420) [inline src/app/components/work-item-list-entry/work-item-list-entry.component.html]: Confused by this - why have the child element as `pull-left` when the parent element is already set to `pull-left`? Unless you are setting things to align to the right, they should default to left alignment.

File: src/app/components/work-item-list-entry/work-item-list-entry.component.html (hunk 4 above)
Comment (SMahil) [reply src/app/components/work-item-list-entry/work-item-list-entry.component.html]: margin-h-10 refers to margin left and right 10px. Its in formulas.less

File: src/app/components/work-item-list-entry/work-item-list-entry.component.html (hunk 2 above)
Comment (SMahil) [reply src/app/components/work-item-list-entry/work-item-list-entry.component.html]: .f8-wi__list-description is generic class for three divs. the last div width is varying from other two and rest of the css is same

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2319-3 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2319-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2319-5 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2319-fabric8-planner.badger.fabric8.io

**Pull request merged**